samples/azure-quantum/shorter-quantum-circuits/shorter-quantum-circuits-dataset.ipynb

```diff
@@ -2675,7 +2673,7 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "Above results are used in [Resource Estimation](../resource-estimation/README.md) notebooks and in the related [paper](https://arxiv.org/pdf/2211.07629.pdf)."
+    "Above results are used in [Resource Estimation](https://github.com/microsoft/Quantum/blob/a9be5a763e6724d0edf3347d6966ea80bf87c141/samples/azure-quantum/resource-estimation/README.md) notebooks and in the related [paper](https://arxiv.org/pdf/2211.07629.pdf)."
```

Comment:
Let's use the main branch, rather than hardcode githashes here, since I doubt we will remember to update this later